docs/content/examples/tutorials/mailbox-compression.md

```diff
@@ -0,0 +1,142 @@
+---
+title: 'Tutorials | Mailbox Compression'
+---
+
+## Overview
+
+Compression allows all messages compressed or not to be READ, and/or compressed on save, regardless of the Algorithm chosen for write. Dovecot message compression plugin supports a variety of algorithms, please review them and the options available at:
+
+* Dovecot 2.3- / DMS 15-: [dovecot-zlib](https://doc.dovecot.org/2.3/configuration_manual/zlib_plugin/)
+* Dovecot 2.4+ / DMS 16+: [mail-compress](https://doc.dovecot.org/2.4.1/core/plugins/mail_compress.html)
+
+Using `zstd` method seems to be the best and faster algorithm to date, according to a recent [mail_compress-benchmark](https://github.com/dovecot/documentation/edit/main/docs/core/plugins/mail_compress.md).
+
+While compression can be memory intensive for large messages, you can configure the plugin to limit the amount of memory consumed by the workers. Examples shown mention the `vsz_limit` setting to limit to 1GB RAM for instance.
+
+## Setup
+
+### Dovecot < 2.4 / DMS < 16
+
+To configure  as a dovecot plugin, create a file at `docker-data/dms/config/dovecot/12-compress-2.3.conf` and place the following in it:
+
+```
+mail_plugins = $mail_plugins zlib
+
+# Enable WRITE compression on saving messages:
+plugin {
+  # zlib_save = gz
+  # zlib_save_level = 1
+  zlib_save = zstd
+}
+
+# Enable READ compression for IMAP:
+# Enable Zlib for imap
+protocol imap {
+  mail_plugins = $mail_plugins zlib
+}
+
+# Enable READ compression for POP3:
+protocol pop3 {
+  mail_plugins = $mail_plugins zlib
+}
+
```

Comment:
As noted, these are redundant, they duplicate the `zlib` addition inherited from the global `mail_plugins` setting (_which is the reason for needing the `12-` prefix for the filename in the first place_).

```suggestion
```